demo/src/main/java/com/example/demo/model/DTO/PersonDTO.java

```diff
@@ -0,0 +1,38 @@
+package com.example.demo.model.DTO;
+
+import lombok.Builder;
+import lombok.Data;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.List;
+
+@Data
+@Builder
+public class PersonDTO {
+
+    @NotBlank(message = "Invalid name: Empty name")
```

Comment:
You don't need both checks, because @NotBlank will return false if the string value is null. You can check https://www.baeldung.com/java-bean-validation-not-null-empty-blank to see best use for each of them.